subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_resources.h

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* Use the timer instance ID, not NRF_TIMERx directly, so that it can be checked
+ * in radio_nrf5_ppi.h by the preprocessor.
+ */
+#if !defined(CONFIG_BT_CTLR_TIFS_HW)
```

Comment:
Suggest to reverse order here, i.e. do 

```c
#if defined(CONFIG_BT_CTLR_TIFS_HW)
...
#else
...
#endif /* CONFIG_BT_CTLR_TIFS_HW */
```

as IMO it's nicer to have the true branch first

Reply:
done